pkg/cmd/chat/channel/channel_test.go

```diff
@@ -111,5 +111,5 @@ func TestListChannel(t *testing.T) {
 	cmd.SetArgs([]string{"list-channels", "-t", "messaging", "-l", "1"})
 	_, err := cmd.ExecuteC()
 	require.NoError(t, err)
-	require.Contains(t, cmd.OutOrStdout().(*bytes.Buffer).String(), "messaging")
+	require.Contains(t, cmd.OutOrStdout().(*bytes.Buffer).String(), "Channels")
```

Comment:
This test doesn't cover if the rendering is broken right? Shouldn't we add more tests for it?

Reply:
Fixed the test